src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -215,6 +215,17 @@ public void validateArguments() {
                             Utils.join(", ", duplicateUserFilterNames)));
         }
 
+        // throw if any disabled filter is duplicated
+        final Set<String> uniqueDisabledFiltersNames = new HashSet<>();
+        final Set<String> duplicateDisabledFilterNames = disableFilters
+                .stream().filter(name -> !uniqueDisabledFiltersNames.add(name))
+                .collect(Collectors.toSet());
```

Comment:
Extract to a utility method in `Utils` that takes a `List<T>` and returns a `Set<T>` containing the duplicated items.